tests/unit_tests/test_aerie_host.py

```diff
@@ -0,0 +1,106 @@
+from typing import Dict
+import pytest
+import requests
+
+from aerie_cli.aerie_host import AerieHost, COMPATIBLE_AERIE_VERSIONS, AerieJWT
+
+
+class MockJWT:
+    def __init__(self, *args, **kwargs):
+        self.default_role = 'viewer'
+
+class MockResponse:
+    def __init__(self, json: Dict, text: str = None, ok: bool = True) -> None:
+        self.json_data = json
+        self.text = text
+        self.ok = ok
+
+    def json(self) -> Dict:
+        if self.json_data is None:
+            raise requests.exceptions.JSONDecodeError("", "", 0)
+        return self.json_data
+
+
+class MockSession:
+
+    def __init__(self, mock_response: MockResponse) -> None:
+        self.mock_response = mock_response
+
+    def get(self, *args, **kwargs) -> MockResponse:
+        return self.mock_response
+
+    def post(self, *args, **kwargs) -> MockResponse:
+        return self.mock_response
+
+
+def get_mock_aerie_host(json: Dict = None, text: str = None, ok: bool = True) -> AerieHost:
+    mock_response = MockResponse(json, text, ok)
+    mock_session = MockSession(mock_response)
+    return AerieHost("", "", mock_session)
+
+
+def test_check_aerie_version():
+    aerie_host = get_mock_aerie_host(
+        json={"version": COMPATIBLE_AERIE_VERSIONS[0]})
+
+    aerie_host.check_aerie_version()
```

Comment:
Is there no Python equivalent of an `assertDoesNotThrow` for unit tests?

Reply:
Nope - tests that raise an error fail by default (at least with standard lib unittest and pytest), so no need to assert something doesn't raise an error.